### PR TITLE
chore(cketh/ckERC20): proposal to upgrade ledger suite orchestrator and managed canisters

### PR DESCRIPTION
**Pull request by gregorydemay**
Proposal to upgrade all ledger suites managed by the ledger suite orchestrator to the latest version.

### CODE REVIEW

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Upgrade all ledger suites managed by the orchestrator to the latest version to continue the migration towards stable memory.
+
+
+## Upgrade args
+
+```
+git fetch
+git checkout e54d3fa34ded227c885d04e64505fa4b5d564743
+cd rs/ethereum/ledger-suite-orchestrator
+didc encode -d ledger_suite_orchestrator.did -t '(OrchestratorArg)' '(variant { UpgradeArg = record {git_commit_hash = opt "e54d3fa34ded227c885d04e64505fa4b5d564743"; ledger_compressed_wasm_hash = opt "98a7b7391608dc4a554d6964bad24157b6aaf890a05bbaad3fcc92033d9c7b02"; index_compressed_wasm_hash = opt "07dd7a18d047ac41c37be9ea200dc1e0cbe4606bbc737d5dbb89f6e0a6e7450d"; archive_compressed_wasm_hash = opt "3ba6fee3ce3d311eef8df0220ac38a411802ee57c0d073847ef2cf3efa6c60ed"}})' | xxd -r -p | sha256sum
```
Comment (gregorydemay) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:26]: Hash output
```
dc97f53a8c9e15e34b3fe16a69f2621fd29d4d0e0f18687315c53cb533be3af2
```

Comment (THLO) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:26]: Hash output:

```
dc97f53a8c9e15e34b3fe16a69f2621fd29d4d0e0f18687315c53cb533be3af2
```

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+git checkout e54d3fa34ded227c885d04e64505fa4b5d564743
+"./ci/container/build-ic.sh" "--canisters"
+sha256sum ./artifacts/canisters/ic-ledger-suite-orchestrator-canister.wasm.gz
+```
+
+Verify that the hash of the gzipped WASM for the ledger, index and archive match the proposed hash.
+
+```
+git fetch
+git checkout d4ee25b0865e89d3eaac13a60f0016d5e3296b31
```
Comment (THLO) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:99]: Is this the correct git commit hash?
It is not consistent with the upgrade arguments.

Comment (gregorydemay) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:99]: Thanks for double-checking! This is a copy-paste error from the previous proposal!

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md (hunk 1 above)
Comment (THLO) [inline rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:26]: Hashes verified:
$ sha256sum ./artifacts/canisters/ic-icrc1-ledger-u256.wasm.gz
98a7b7391608dc4a554d6964bad24157b6aaf890a05bbaad3fcc92033d9c7b02
$ sha256sum ./artifacts/canisters/ic-icrc1-index-ng-u256.wasm.gz
07dd7a18d047ac41c37be9ea200dc1e0cbe4606bbc737d5dbb89f6e0a6e7450d
$ sha256sum ./artifacts/canisters/ic-icrc1-archive-u256.wasm.gz
3ba6fee3ce3d311eef8df0220ac38a411802ee57c0d073847ef2cf3efa6c60ed

**Review by THLO — APPROVED** [review]

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md (hunk 1 above)
Comment (ninegua) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:26]: Hashes verified!

Comment (ninegua) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:26]: `dc97f53a8c9e15e34b3fe16a69f2621fd29d4d0e0f18687315c53cb533be3af2`

File: rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
+# Proposal to upgrade the ledger suite orchestrator canister
+
+Repository: `https://github.com/dfinity/ic.git`
+
+Git hash: `e54d3fa34ded227c885d04e64505fa4b5d564743`
+
+New compressed Wasm hash: `b72d9ce3a174b3f2a086cf60225177ae1b09036008e413592d0b17ad599a8614`
```
[the remark this answers is not in the document]
Comment (ninegua) [reply rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_10_25.md:7]: Verified

**Review by ninegua — APPROVED** [review]

**Pull request merged**